SPECS/mstflint/mstflint.spec

```diff
@@ -15,6 +15,8 @@ BuildRequires:  automake
 BuildRequires:  boost-devel
 BuildRequires:  gcc
 BuildRequires:  gcc-c++
+BuildRequires:  kernel-devel
```

Comment:
Since we're using this to build a kernel module, let's be explicit about the version we're using with `BuildRequires: kernel-devel = <version>-<release>`. One idea might be to follow what we do with livepatches, which also deal with kernel modules. At the top of their specs we define:
```
%define kernel_version_release 5.15.87.1-1.cm2
```

and then use that macro in `BuildRequires: kernel-devel = %{kernel_version_release}`.

Reply:
Following our discussion, we are not going for this option.